crates/sncast/src/starknet_commands/declare.rs

```diff
@@ -47,19 +47,24 @@ pub async fn declare(
     nonce: Option<FieldElement>,
     build_config: BuildConfig,
     wait_config: WaitForTx,
+    make_scarb_quiet: bool,
```

Comment:
As we discussed pleace create an issue to drop scarb compilation from declare.

Reply:
https://github.com/foundry-rs/starknet-foundry/issues/1474